lib/reconcile/astarte_generic_api.go

```diff
@@ -257,11 +257,12 @@ func getAstarteGenericAPIEnvVars(deploymentName string, cr *apiv1alpha1.Astarte,
 			Value: getCassandraNodes(cr),
 		})
 
+		// Append Cassandra connection env vars only if version >= 1.0.0
```

Comment:
It's no surprise the CI fails. You always want to add the `nodes` env to any version, otherwise services simply won't connect